### PR TITLE
Improve dark mode code colours Fixes #2377

### LINKED ISSUE #2377: Improve dark mode code colours
Our dark mode isn't great for `<code>...</code>` (i.e., inline) and `<pre><code>...</code></pre>` (i.e., block):

Screen Shot 2021-10-21 at 12 30 18 PM

We should probably take a look at how GitHub and other sites render code in dark mode, and see what we can borrow.

### PR DESCRIPTION
**Pull request by japneetsingh035**
## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description
Before:
Screen Shot 2021-11-19 at 10 45 20 AM

After:
Screen Shot 2021-11-19 at 12 35 31 PM

## Checklist

- [X] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [X] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by AmasiaNalbandian** [comment]
Is the color supposed to change only for dark mode? I am seeing the same color scheme for the light mode also: 
image

**Comment by japneetsingh035** [comment]
Yes, I am working on the colour scheme to change on the toggle. However, it would require changing all the previous code for highlight.js. So, I made a draft PR to get some help in development.

**Comment by AmasiaNalbandian** [comment]
> Yes, I am working on the colour scheme to change on the toggle. However, it would require changing all the previous code for highlight.js. So, I made a draft PR to get some help in development.

Can we try something like this? 
https://stackoverflow.com/questions/35762351/correct-way-to-handle-conditional-styling-in-react

Make a conditional in the div for the stylesheet. 
What we can also do it make a variable to toggle the state somewhere [here](https://github.com/Seneca-CDOT/telescope/blob/bff7a86cd25076fee1a7ffa4e4efe9c8f5c16171/src/api/status/public/assets/js/material-dashboard.js#L688), and use this to do the conditional stylesheet.

**Comment by AmasiaNalbandian** [comment]
This is a better example using the link stylesheets: 
https://stackoverflow.com/questions/43971507/if-else-jsx-using-link-in-react

**Review by humphd — CHANGES_REQUESTED** [review]
See https://github.com/Seneca-CDOT/telescope/blob/29951b7b60fa33d83e14b05f24d4a4c2722f9ef7/src/web/src/styles/globals.css#L5

We're going to want to have two stylesheets, both from node_modules vs. a cdn, and you will pick the one at runtime to use.

Also, `src/api/status*` isn't the same code as what you're changing for posts.

File: src/web/src/styles/telescope-post-content.css [hunk 1]
```diff
@@ -106,8 +106,8 @@
   overflow-wrap: unset;
   word-wrap: none;
   font-family: Menlo, Consolas, Monaco, 'Liberation Mono', 'Lucida Console', monospace;
-  background: #eff0f1;
-  color: #242424;
+  background: rgb(14, 17, 22);
```
Comment (PedroFonsecaDEV) [inline src/web/src/styles/telescope-post-content.css:109]: Hex values, please.

**Comment by humphd** [comment]
What's the status of this?

**Comment by japneetsingh035** [comment]
> What's the status of this?

Well, we can close this PR and will make a new one, once I am done - still working on enabling the dark mode colour.

**Pull request closed without merging**